src/ports/postgres/modules/utilities/debug.py_in

```diff
@@ -0,0 +1,142 @@
+import plpy as plpy_orig
+import time
+from deep_learning.madlib_keras_model_selection import ModelSelectionSchema
+from deep_learning.madlib_keras_helper import DISTRIBUTION_KEY_COLNAME
+
+mst_key_col = ModelSelectionSchema.MST_KEY
+dist_key_col = DISTRIBUTION_KEY_COLNAME
+
+start_times = dict()
+timings_enabled = False
```

Comment:
I am not clear on how this, or the other global variables in this file, like `mst_keys_enabled`, are used. I don't see a statement that overrides them and they are not a part of a class so the caller cannot manipulate them either.

I assume the dev just changes these by hand to enable debug for every instance of the function call, which would be an OK use case I guess.